guide/release-notes/v3.0.10.adoc

```diff
@@ -0,0 +1,19 @@
+= Version 3.0.10
+[horizontal]
+Release date:: 2021-05-03
+
+== Changes to BIS document
+
+*Peppol logo updated to new design [POACC-375]
+
+== Changes to support documents
+
+
+== Changes to code lists and validation artefacts
+* Updated some national rules which was using unusual characters in the rule statements, no functional changes. [POACC-398]
+
+*ICD code list details edited to remove contact information [POACC-388]
```

Comment:
Not only contact information has been changed. Maybe further update the release notes to properly reflect the icd.xml change?